public/planning/poste/fonctions.php

```diff
@@ -152,7 +152,7 @@ function cellule_poste($date, $debut, $fin, $colspan, $output, $poste, $site)
         $cellule.="<div id='cellule{$GLOBALS['idCellule']}_$i' class='cellDiv {$classe[$i]} pl-cellule-perso-{$resultats[$i]['perso_id']}' {$color[$i]} data-perso-id='{$resultats[$i]['perso_id']}'>{$resultats[$i]['text']}</div>";
     }
 
-    $cellule .= '<a class="pl-icon arrow-right" href="#"></a>';
+    $cellule .= '<a class="pl-icon arrow-right"></a>';
```

Comment:
I would perhaps add role="button" for accessibility:
`$cellule .= '<a class="pl-icon arrow-right" role="button"></a>';`
See https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/Roles/button_role